sw/airborne/modules/computer_vision/lib/vision/fast_rosten.c

```diff
@@ -46,13 +46,12 @@ static void fast_make_offsets(int32_t *pixel, uint16_t row_stride, uint8_t pixel
  * @param[out] *num_corners The amount of corners found
  * @return The corners found
  */
-struct point_t *fast9_detect(struct image_t *img, uint8_t threshold, uint16_t min_dist, uint16_t x_padding, uint16_t y_padding, uint16_t *num_corners) {
+struct point_t *fast9_detect(struct image_t *img, uint8_t threshold, uint16_t min_dist, uint16_t x_padding, uint16_t y_padding, uint16_t *num_corners, uint16_t *corners_size,struct point_t *ret_corners) {
   uint32_t corner_cnt = 0;
```

Comment:
if corners_size is the number of corners and not the size of the corner this is a bad name...